equinix/fabric_port_read_schema.go

```diff
@@ -0,0 +1,251 @@
+package equinix
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func readPortDeviseSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Port Name",
+		},
+		"redundancy": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Port Device Redundancy",
+			Elem: &schema.Resource{
+				Schema: readRedundancySch(),
+			},
+		},
+	}
+}
+
+func readPortOperationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"operational_status": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Port Type",
+		},
+		"connection_count": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Total number of current connections",
+		},
+		"op_status_changed_at": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Date and time at which port availability changed",
+		},
+	}
+}
+
+func readFabricPortTether() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"cross_onnect_id": {
```

Comment:
```suggestion
		"cross_connect_id": {
```

I assume this is a typo